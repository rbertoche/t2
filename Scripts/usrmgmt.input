# Cria usuarios e loga e desloga em todos eles
register a
register b
register c
login c
logout
login b
logout
login v
login a
logout

# Loga e deleta um usario no meio

login b
delme
login x
login a
logout
login c
logout

