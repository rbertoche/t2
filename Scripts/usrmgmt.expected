Novo usuario criado.
Novo usuario criado.
Novo usuario criado.
bye c
bye b
login as 'v' failed
bye a
bye b
Usuario excluido com sucesso.
login as 'x' failed
bye a
bye c

